### PR TITLE
Boost log -> spdlog

### PR DESCRIPTION
**Pull request by msgmaxim**

### CODE REVIEW

File: httpserver/main.cpp [hunk 1]
```diff
@@ -7,6 +7,10 @@
 #include "swarm.h"
 #include "version.h"
 
+#include "spdlog/spdlog.h"
+#include "spdlog/sinks/stdout_color_sinks.h"
```
Comment (BeaudanBrown) [inline httpserver/main.cpp]: These 2 are already included below

Comment (msgmaxim) [reply httpserver/main.cpp]: removed

File: common/src/common.h [hunk 2]
```diff
@@ -9,18 +9,23 @@
...
 namespace fs = boost::filesystem;
 
 // clang-format off
-#define LOKI_LOG(lvl)\
-    BOOST_LOG_STREAM_WITH_PARAMS(logging::trivial::logger::get(),\
-        (set_get_attrib("File", fs::path(__FILE__).filename().string())) \
-        (set_get_attrib("Line", __LINE__)) \
-        (set_get_attrib("Func", std::string(__FUNCTION__))) \
-        (logging::keywords::severity = logging::trivial::lvl))
+// #define LOKI_LOG(lvl)\
```
Comment (BeaudanBrown) [inline common/src/common.h]: Can we scrap these comments?

Comment (msgmaxim) [reply common/src/common.h]: done

**Pull request merged**